cmd/aries-agentd/startcmd/start.go

```diff
@@ -48,15 +48,31 @@ const (
 	// AgentDBPathFlagUsage is the flag usage text for the database path command line argument.
 	AgentDBPathFlagUsage = "Path to database"
 
-	// MissingHostErrorMessage is the error message shown when the user provides a blank host argument.
-	MissingHostErrorMessage = "Unable to start aries agentd, host not provided"
+	// AgentWebhookFlagName is the flag name for the webhook command line argument.
+	AgentWebhookFlagName = "webhook-url"
 
-	// MissingInboundHostErrorMessage is the error message shown when the user provides a blank inbound host argument.
-	MissingInboundHostErrorMessage = "Unable to start aries agentd, HTTP Inbound transport host not provided"
+	// AgentWebhookFlagShorthand is the flag shorthand name for the webhook command line argument.
+	AgentWebhookFlagShorthand = "w"
+
+	// AgentWebhookFlagUsage is the flag usage text for the webhook command line argument.
+	AgentWebhookFlagUsage = "URL to send notifications to." +
+		" This flag can be repeated, allowing for multiple listeners."
 )
 
+// ErrMissingHost is the error when the user provides a blank host argument.
+var ErrMissingHost = errors.New("unable to start aries agentd, host not provided")
+
+// ErrMissingInboundHost is the error when the user provides a blank inbound host argument.
+var ErrMissingInboundHost = errors.New("unable to start aries agentd, HTTP Inbound transport host not provided")
```

Comment:
It would be great if we took this opportunity to fix this documentation: this "inbound host" is really the listen address for the DIDComm service served over the HTTP transport.

Reply:
What should be changed exactly?